src/api/routers/participants/participantsRouter.ts

```diff
@@ -136,8 +140,10 @@ export function createParticipantsRouter() {
   participantsRouter.get('/:participantId/auditTrail', handleAuditTrail);
 
   participantsRouter.get('/:participantId/users', handleGetParticipantUsers);
+  participantsRouter.get('/:participantId/:userId', handleGetUserRolesForCurrentParticipant);
```

Comment:
let's change this endpoint so that it's less confusing. e.g. an example call would be `http://localhost:3000/api/participants/8/3` - it is hard to know what the 8 and the 3 mean. Whereas the other endpoints in the `participantUsersRouter` are of the form `/:participantId/users/:userId` e.g. http://localhost:3000/api/participants/8/users/3`. This may require moving the endpoint to `participantUsersRouter.ts`